StarterProject/README.md

```diff
@@ -0,0 +1,83 @@
+
+****************************************************************************************************************
+
+STARTER PROJECT
+
+
+This starter project is used to give you the user a short introduction into webdriver.io and to help you to get up and running as soon as possible.
+
+This starter project contains a suite of tests that accomplishes the following actions:
+ - Navigate to http://www.clubmonaco.ca/
+ - Check if the email modal exists
+ - Close the email modal
+ - Search for a prouct with the search
+ - Ensure that related product flyout is present
+ - Navigate to a product page
+ - View the assets in the product page
+
+****************************************************************************************************************
+
+PREREQUISITES
+
+
+ Encountered a problem with reinstallig webdriver.io with npm install so this workaround of installing global packages instead.
+ Install all prerequisites with ./install-global-npm.sh 
+
+ 
+This project depends on the user having installed npm version 5.7.1 go to https://www.npmjs.com/ if you need to more information 
+
+and just in case you dont have node v9.3.0 installed, please walk over to https://nodejs.org/en/ and install it first
```

Comment:
I think this can be removed -- wdio should work with a bunch of older node versions as well